scripts/generators/es_template.py

```diff
@@ -286,30 +269,3 @@ def default_mapping_settings():
             }
         ]
     }
-
-
-def es6_type_fallback(mappings):
-    """
-    Visits each leaf in mappings object and fallback to an
-    Elasticsearch 6.x supported type.
-
-    Since a field like `wildcard` won't have the same defaults as
-    a `keyword` field, we must add any missing defaults.
-    """
-
-    for (name, details) in mappings.items():
-        if 'type' in details:
-            fallback_type = TYPE_FALLBACKS.get(details['type'])
-            if fallback_type:
-                mappings[name]['type'] = fallback_type
-                field_or_multi_field_datatype_defaults(mappings[name])
-        # support multi-fields
-        if 'fields' in details:
-            # potentially multiple multi-fields
-            for field_name, field_value in details['fields'].items():
-                fallback_type = TYPE_FALLBACKS.get(field_value['type'])
-                if fallback_type:
-                    mappings[name]['fields'][field_name]['type'] = fallback_type
-                    field_or_multi_field_datatype_defaults(mappings[name]['fields'][field_name])
```

Comment:
This import from `schema.cleaner` might be removable `field_or_multi_field_datatype_defaults` now.